src/http/adapter.js

```diff
@@ -132,21 +132,44 @@ function getActionMetadata( state, resource, actionName, action, meta, resources
 	};
 }
 
+function getHandler( handle ) {
+	if( _.isFunction( handle ) ) {
+		return handle;
+	} else {
+		var list = _.map( handle, function( option ) {
+			if( _.isFunction( option.condition ) ) {
+				return option;
+			} else {
```

Comment:
@digitalBush perhaps? I _could_ do that and then have an else that yells at you? It's fun to yell!